packages/odf/components/mcg/ObjectBucket.tsx

```diff
@@ -102,6 +103,9 @@ const OBRow: React.FC<RowProps<K8sResourceKind, CustomData>> = ({
             resource: obj,
             resourceModel: NooBaaObjectBucketModel,
           }}
+          hoverMessage={t(
```

Comment:
```suggestion
          terminatingTooltip={t(
```
let's keep implementation/terminology same between console/odf console both...